README.md

```diff
@@ -1,23 +1,18 @@
 [![Build Status](https://travis-ci.org/rapid7/r7insight_node.svg?branch=master)](https://travis-ci.org/rapid7/r7insight_node)
 
-# r7insight_node: InsightOps Client
+# r7insight_node: Insight Platform Client
 
-Allows you to send logs to your [InsightOps](https://www.rapid7.com/solutions/it-operations/)
+Allows you to send logs to the [Insight Platform](https://www.rapid7.com/products/)
 (or Logentries) account from Node.js.
 
-> It might work with Browserify, too, but you would need to use shims for net
-> and tls. Such shims do exist, based on forge, but I haven’t tested it. There’s
-> a seperate client intended for use in the browser though, called
-> [le_js](https://www.npmjs.com/package/le_js), which uses http and is optimized
-> for browser-specific logging needs.
-
-What is now "r7insight_node" was previously "logentries-client"; users of r7insight_node
-versions before 1.0.2 should read the sections below that detail differences if
-they wish to update.
+There’s a separate client intended for use in the browser, called
+[le_js](https://www.npmjs.com/package/le_js), which uses http and is optimized
```

Comment:
Is this not deprecated, like most "le" libraries?

Reply:
yup, i'll replace it with r7insight_js